hls4ml/templates/vivado/nnet_utils/nnet_merge.h

```diff
@@ -31,6 +31,12 @@ struct merge_config
     static const unsigned n_elem = 10;
 };
 
+
+struct dot_config {
+    static const unsigned n_elem = 10;
+    static const unsigned n_elem_out = 1;
+};
```

Comment:
We're missing a `reuse_factor` field here, since we are using multiplication in implementation.